src/qt/rpcconsole.cpp

```diff
@@ -414,7 +415,32 @@ void RPCExecutor::request(const QString &command, const WalletModel* wallet_mode
     {
         std::string result;
         std::string executableCommand = command.toStdString() + "\n";
-
+        std::regex validRegex("generate\\s*[\\(]?\\s*(\\d+)?\\s*[,]?\\s*(\\d+)?\\s*[\\)]?\\s*");
```

Comment:
This is pretty complicated regex. Would it be easier to split `executableCommand` into tokens separated by whitespace?

Reply:
See https://github.com/bitcoin/bitcoin/blob/master/src/bitcoin-cli.cpp#L679-L684.